README.md

```diff
@@ -192,6 +193,7 @@ sd-dev-proxy sd-dev allow
 ```
 
 9. Verify qrexec communication between `sd-dev-proxy` and `sd-dev` is set up properly.
+
```

Comment:
formatting of bash code and step a wasn't showing up properly

Reply:
Fixes step a, but note that the bash formatting is still borked: https://github.com/freedomofpress/securedrop-sdk/blob/2141af5eb7e87f89bda616c6e09b76337b8811dc/README.md